docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -32,6 +32,7 @@
   * [Sharding-JDBC](https://github.com/shardingjdbc/sharding-jdbc) 1.5.x
   * [ShardingSphere](https://github.com/apache/incubator-shardingsphere) 3.0.0, 4.0.0-RC1
   * PostgreSQL Driver 8.x, 9.x, 42.x
+  * Cassandra Driver 3.7.0-3.7.2
```

Comment:
Cassandra is already in the NoSQL section.

Reply:
ACK